tools/ratbagd.py

```diff
@@ -111,7 +111,11 @@ def __init__(self, interface, object_path):
         GObject.GObject.__init__(self)
 
         if _RatbagdDBus._dbus is None:
-            _RatbagdDBus._dbus = Gio.bus_get_sync(Gio.BusType.SYSTEM, None)
+            try:
+                _RatbagdDBus._dbus = Gio.bus_get_sync(Gio.BusType.SYSTEM, None)
+            except GLib.Error:
+                raise RatbagdDBusUnavailable()
+
             if _RatbagdDBus._dbus is None:
```

Comment:
fwiw, the Is None thing was part of the other PR and is already merged, so I'll keep this local and push it with some other PR